e2e/charts/logging-operator-logging/templates/logging.yaml

```diff
@@ -5,69 +5,76 @@ metadata:
   labels:
 {{ include "logging-operator-logging.labels" . | indent 4 }}
 spec:
-  {{- with .Values.enableRecreateWorkloadOnImmutableFieldChange }}
-  enableRecreateWorkloadOnImmutableFieldChange: {{ . }}
-  {{- end}}
   {{- with .Values.loggingRef }}
   loggingRef: {{ . }}
   {{- end }}
   {{- with .Values.flowConfigCheckDisabled }}
   flowConfigCheckDisabled: {{ . }}
   {{- end }}
-  {{- with .Values.flowConfigOverride }}
-  flowConfigOverride: {{ . }}
-  {{- end }}
-  {{- with .Values.allowClusterResourcesFromAllNamespaces }}
-  allowClusterResourcesFromAllNamespaces: {{ . }}
-  {{- end }}
   {{- with .Values.skipInvalidResources }}
   skipInvalidResources: {{ . }}
   {{- end }}
-  {{- if .Values.watchNamespaces }}
-  watchNamespaces:
-{{ toYaml .Values.watchNamespaces | indent 4 }}
-  {{- end }}
-  clusterDomain: {{ .Values.clusterDomain }}
-  controlNamespace: {{ .Values.controlNamespace | default .Release.Namespace }}
-  {{- if .Values.defaultFlow }}
-  defaultFlow:
-{{ toYaml .Values.defaultFlow | indent 4 }}
-  {{- end }}
-  {{- if .Values.globalFilters }}
-  globalFilters:
-{{ toYaml .Values.globalFilters | indent 4 }}
+  {{- with .Values.flowConfigOverride }}
+  flowConfigOverride: {{ . }}
   {{- end }}
-  {{- if or .Values.tls.enabled .Values.fluentd }}
-  fluentd:
+  {{- if or .Values.tls.enabled .Values.fluentbit }}
+  fluentbit:
     {{- if .Values.tls.enabled }}
     tls:
       enabled: true
-      secretName: {{ .Values.tls.fluentdSecretName | default (printf "%s-%s" (include "logging-operator-logging.name" . ) "fluentd-tls" ) }}
+      secretName: {{ .Values.tls.fluentbitSecretName | default (printf "%s-%s" (include "logging-operator-logging.name" . ) "fluentbit-tls" ) }}
       sharedKey: "{{ .Values.tls.sharedKey }}"
     {{- end }}
-    {{- if .Values.fluentd }}
-{{ toYaml .Values.fluentd | indent 4 }}
+    {{- with .Values.fluentbit }}
+    {{- toYaml . | nindent 4 }}
     {{- end }}
   {{- else }}
-  fluentd: {}
+  fluentbit: {}
   {{- end }}
-  {{- if or .Values.tls.enabled .Values.fluentbit }}
-  fluentbit:
+  {{- if or .Values.tls.enabled .Values.fluentd }}
+  fluentd:
     {{- if .Values.tls.enabled }}
     tls:
       enabled: true
-      secretName: {{ .Values.tls.fluentbitSecretName | default (printf "%s-%s" (include "logging-operator-logging.name" . ) "fluentbit-tls" ) }}
+      secretName: {{ .Values.tls.fluentdSecretName | default (printf "%s-%s" (include "logging-operator-logging.name" . ) "fluentd-tls" ) }}
       sharedKey: "{{ .Values.tls.sharedKey }}"
     {{- end }}
-    {{- if .Values.fluentbit }}
-{{ toYaml .Values.fluentbit | indent 4 }}
+    {{- with .Values.fluentd }}
+    {{- toYaml . | nindent 4 }}
     {{- end }}
   {{- else }}
-  fluentbit: {}
+  fluentd: {}
+  {{- end }}
```

Comment:
this is incorrect I beleive (and it was incorrect previously as well)

this way we cannot disable fluentd completely, although that is a totally valid scenario as well (when syslog-ng replaces fluentd)

I can't add a suggestion due to the sorting but it should be this:
```
  {{- else if .Values.fluentd }}
  fluentd: {}
  {{- end }}
```